src/index.js

```diff
@@ -77,6 +109,11 @@ function convertToString (input) {
 module.exports = {
   multihash: isMultihash,
   cid: isCID,
+  base32cid: (cid) => (isMultibase(cid) === 'base32' && isCID(cid)),
+  ipfsSubdomain: (url) => isIpfs(url, fqdnPattern, fqdnProtocolMatch, fqdnHashMatch),
+  ipnsSubdomain: (url) => isIpns(url, fqdnPattern, fqdnProtocolMatch, fqdnHashMatch),
+  subdomain: (url) => (isIpfs(url, fqdnPattern, fqdnProtocolMatch, fqdnHashMatch) || isIpns(url, fqdnPattern, fqdnProtocolMatch, fqdnHashMatch)),
```

Comment:
As you're calling `(ipfsSubdomain || ipnsSubdomain)`, you should declare them before the `module.exports`. That way if any of those change, you won't need to change that code in two places.

To make things clearer:

```js
const ipfsSubdomain = (url) => isIpfs(url, fqdnPattern, fqdnProtocolMatch, fqdnHashMatch)
const ipnsSubdomain = (url) => isIpns(url, fqdnPattern, fqdnProtocolMatch, fqdnHashMatch)

module.exports = {
	multihash: isMultihash,
	cid: isCID,
	base32cid: (cid) => (isMultibase(cid) === 'base32' && isCID(cid)),
	ipfsSubdomain: ipfsSubdomain,
	ipnsSubdomain: ipnsSubdomain,
	subdomain: (url) => (ipfsSubdomain(url) || ipnsSubdomain(url)),
	...
}
```